docs/reference/search/validate.asciidoc

```diff
@@ -1,8 +1,74 @@
 [[search-validate]]
 === Validate API
 
-The validate API allows a user to validate a potentially expensive query
-without executing it. We'll use the following test data to explain _validate:
+Validates a potentially expensive query without executing it.
+
+[source,console]
+--------------------------------------------------
+GET twitter/_validate/query?q=user:foo
+--------------------------------------------------
+
+
+[[search-validate-api-request]]
+==== {api-request-title}
+
+`GET /<index>/_validate/<query>`
+
+
+[[search-validate-api-desc]]
+==== {api-description-title}
+
+The validate API allows you to validate a potentially expensive query
+without executing it. The query can be sent either as a path parameter or in the 
+request body.
+
+
+[[search-validate-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
+
+`query`::
```

Comment:
This should probably be moved to `common-parms.asciidoc`.

Reply:
https://github.com/elastic/elasticsearch/pull/46377/ adds the parameter.